src/status_im/common/password_with_hint/view.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.common.password-with-hint.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [status-im.common.password-with-hint.style :as style]))
+
+(defn view
+  [{{:keys [text status shown?]} :hint :as input-props}]
+  [rn/view
```

Comment:
Could it be a fragment instead of `view`?

Reply:
Yeah good idea! I'll change it to a fragment 👍